docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -225,7 +225,7 @@ Mbed CLI supports a setting for each toolchain path:
 | --------- | --------- | ---------| --------- |
 | Arm Compiler 5 | `C:/Program Files/ARM_Compiler_5.06u5/bin/armcc` | `ARM_PATH` | `C:/Program Files/ARM_Compiler_5.06u5` |
```

Comment:
@cmonr @timurh01  - ARM Compiler 5 supported and in CI is ARM_Compiler_5.06u5 or ARM_Compiler_5.06u6